### PR TITLE
feat: eclipse/che#18831 - Add option for updateBaseImages.sh to enforce PR creation

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add option to enforce creation of PR when updating base images, to prevent unwanted direct push to target main branch

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18831

### CODE REVIEW

File: product/updateBaseImages.sh [hunk 1]
```diff
@@ -261,24 +267,19 @@ for d in $(find ${WORKDIR}/ -maxdepth ${MAXDEPTH} -name ${DOCKERFILE} | sort -r)
...
-											hub pull-request -f -m "${lastCommitComment}
-
-${lastCommitComment}" -b "${BRANCHUSED}" -h "${PR_BRANCH}" "${OPENBROWSERFLAG}" || true 
-										else
-											echo "# Warning: hub is required to generate pull requests. See https://hub.github.com/ to install it."
-											echo -n "# To manually create a pull request, go here: "
-											git config --get remote.origin.url | sed -r -e "s#:#/#" -e "s#git@#https://#" -e "s#\.git#/tree/${PR_BRANCH}/#"
+								git commit -s -m "chore: Update from ${URL} to ${FROMPREFIX}:${LATESTTAG}" ${DOCKERFILE}
+								if [[ ${dopronly} -eq 1]]; then
+									if [[ ${dopush} -eq 1 ]]; then
```
Comment (nickboldt) [inline product/updateBaseImages.sh:272]: remove this second if check; doesn't really mean anything here.

File: product/updateBaseImages.sh [hunk 2]
```diff
@@ -109,6 +111,7 @@ while [[ "$#" -gt 0 ]]; do
     '-s') buildCommand="rhpkg container-build --scratch"; shift 0;;
     '-n'|'--nocommit'|'--no-commit') docommit=0; dopush=0; shift 0;;
     '-p'|'--nopush'|'--no-push') dopush=0; shift 0;;
+	'-pr'|'--pronly'|'--pr-only') dopronly=1; shift 0;;
```
Comment (nickboldt) [inline product/updateBaseImages.sh:114]: don't need 3 flags for this. just use `--pr`

File: product/updateBaseImages.sh (hunk 1 above)
Comment (mkuznyetsov) [reply product/updateBaseImages.sh]: yeah, dunno, maybe no push + pr mode would be used for "testing" purposes?

File: product/updateBaseImages.sh (hunk 2 above)
Comment (mkuznyetsov) [reply product/updateBaseImages.sh]: ok, thx

File: product/updateBaseImages.sh (hunk 1 above)
Comment (nickboldt) [reply product/updateBaseImages.sh]: so you want a PR generated but without generating the PR? that would essentially make the whole if-block do nothing

**Pull request merged**